test/channels/retro_channel_test.exs

```diff
@@ -46,13 +46,6 @@ defmodule RemoteRetro.RetroChannelTest do
 
   describe "pushing `proceed_to_next_stage` with a stage of 'action-item-distribution'" do
     setup [:join_the_retro_channel]
-
```

Comment:
@vanderhoop The main reason we deleted the test was because the modifications to emails.exs broke the test and it would have been a lot of work to get the test passing again with little to no benefit